### ISSUE TITLE
SVG max-width

### ISSUE THREAD

**Issue by Lumelomser** [uikit/uikit#3547]
### UIkit version

v3.0.0-rc.12

### Reproduction Link

https://codepen.io/anon/pen/LJZWKP

### Steps to reproduce

Open reproduction link.
Remove the SVG style in the CSS-Section of the editor (or toggle it in Dev-Tools), so the current style of UIkit will be used.

### What is Expected?

Icons expand the first table cell width, depending on sum of icons.

### What is actually happening?

Icons don't expand the first table cell width.
The second cell flows into the first cell.

Issue caused by the following commit:
https://github.com/uikit/uikit/commit/66c0d9b96d9d96703e37c2eb9cd6ef50fa9bb5d9

**Comment by saschadube** [comment]
Thanks for reporting! 😃 I've got three things:

1. The commit https://github.com/uikit/uikit/commit/66c0d9b96d9d96703e37c2eb9cd6ef50fa9bb5d9 fixed inline SVG images (`<svg>`) not being responsive like normal images (`<img>`). This if fine and a must have.

2. The icon in the Iconnav are not shrinking like images in the Thumbnav would do. I fixed it with this commit https://github.com/uikit/uikit/commit/5d4bc8f043ed0265de7bcf3e59f2aff5ec4304e9. In your example the icons should now shrink without the `max-width`.

3. To make icons in the Iconnav (or images in the Thumbnav) not shrink, add `uk-preserve-width` to the `ul` element. See https://codepen.io/anon/pen/ZMOrEq
